src/arch/wasm32/switch_stacks.s

```diff
@@ -0,0 +1,36 @@
+	.text
+	.section	.text.__stacker_switch_stacks,"",@
+
+	.functype	typeindex0 (i32) -> ()
+
+	.globl	__stacker_switch_stacks
+	.type	__stacker_switch_stacks,@function
+__stacker_switch_stacks:                # @__stacker_switch_stacks
+	
+	.functype	__stacker_switch_stacks (i32, i32, i32) -> ()
+	.local  	i32 # temporary storage for the original stack pointer
+# %bb.0:                                # %entry
+	# save current stack pointer
+	get_global	__stack_pointer@GLOBAL
+	set_local	3
```

Comment:
Technically I think you can remove this `set_local` and the `get_local 3` below and this should continue to work as expected?

Reply:
Yes, we can! However, we have to keep `.local i32` because of some bug in clang.